### DISCUSSION TITLE
When is OSD 2.8.0 compatible enhanced tables plugin going to be released?

### DISCUSSION THREAD

**Discussion by richardtruefort** [fbaligand/kibana-enhanced-table#325] [General]
Hello, OpenSearch Dashboards 2.8.0 was released today.

I know it takes a while for plugins to be updated for new version, but do we have an idea of how long it would take for enhanced tables distribution compatible with 2.8.0 to be released?

Thank you

**Comment by fbaligand** [comment]
Hi,
I just published the package for OSD 2.8.0 here:
https://github.com/fbaligand/kibana-enhanced-table/releases/tag/v1.13.3

Actually, since I released a "Enhanced Table version" for OpenSearch Dashboards, I never had breaking changes in new OSD versions that require changes in Enhanced Table plugin. So my code is the same for all OSD versions. 
I hope that it will continue like this as long as possible...

**Reply by richardtruefort** [reply]
Thank you!
